bokeh/document/document.py

```diff
@@ -971,6 +971,10 @@ def validate(self):
             refs = r.references()
             check_integrity(refs)
 
+            # Currently not sure how to access messages inside this function
+            if len(messages['error']) or (len(messages['warning']) and settings.strict()):
```

Comment:
`settings.strict()` corresponds to `BOKEH_STRICT` this needs to be updated to process the new `settings.validation_exceptions()` instead

Reply:
But see note below about changing this name to `validation_level`